docs/src/pages/demos/selection-controls/CustomizedSwitches.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { ChangeEvent } from 'react';
```

Comment:
We are sticking to this convention in the codebase. It reduces the cost of refactoring code. 

```suggestion
import React from 'react';
```